pupil_src/shared_modules/accuracy_visualizer.py

```diff
@@ -371,6 +385,9 @@ def recalculate(self):
             succession_threshold=self.succession_threshold,
         )
 
+        if not results.is_valid:
+            return
```

Comment:
I suggest adding user feedback at this point. The suggestion below is just a copy from the warning above. A warning instead of an info message might be more appropriate. What do you think?
```suggestion
            logger.info(
                "Did not collect enough data to estimate gaze mapping accuracy."
            )
            return
```